packages/zosmf/src/CheckStatus.ts

```diff
@@ -36,6 +36,10 @@ export class CheckStatus {
         return ZosmfRestClient.getExpectJSON(session, infoEndpoint);
     }
 
+    public static async isZosVersionGreaterThan(session: AbstractSession, version: string): Promise<boolean> {
+        return (await CheckStatus.getZosmfInfo(session)).zosmf_version >= version;
```

Comment:
Not sure what will be more valuable here.

If we should compare versions (i.e. `>=`) or if we should return an enum-like value (`ZosmfConstants.Versions.V2R1`, ...)

For our purposes, this works just fine 😋